pvm.py

```diff
@@ -1,18 +1,26 @@
 from datetime import datetime
 import os
+from pathlib import Path
 from pythonosc import dispatcher, osc_server
 import argparse
 from omxplayer.player import OMXPlayer
 import logging
 from logging.handlers import TimedRotatingFileHandler
 import sys
 
+# Place your videos in this folder for autostart
+HOME = str(Path.home()) + "/"  # The home directory, e.g. /home/pi/
+LOG_PATH = HOME + "PVM/log/"
+PEFIX_PATH = HOME + "Videos/"
```

Comment:
interesting strategy here, and it might work just fine, especially if users follow the install instructions and build PVM in the right place. I was wondering, however, if there is a way to know the location of the python file being executed and work up from there... [[1](https://stackoverflow.com/questions/595305/how-do-i-get-the-path-of-the-python-script-i-am-running-in)]
again, don't sweat over this. what you've built here works already.

Reply:
Yes, it's possible (`os.getcwd()`), my concern is with the scenario of autostart. When pvm is launched by the autostart mechanism, the _current directory_ of the shell script being run (`launch.sh`) might not be the same directory that Pyhton files are located.

Reply:
of course, you can't use `os.getcwd()` for that reason, but could you use `os.path.realpath(__file__)`?

Reply:
Interesting idea! I think it would be possible to use `__file__`, but needs to be verified with tests. Probably after everything settles down? So that we could ensure this change won't introduce any regression bugs.

Reply:
cool. I'll trust your judgement here then.